src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/DataElementsCard.test.tsx

```diff
@@ -997,6 +1021,36 @@ describe("DataElementsCard", () => {
     });
   });
 
+  test("Attribute Chip Render", async () => {
+    await waitFor(() =>
+      renderDataElementsCard(
+        "attributes",
+        jest.fn,
+        testDataElements[15],
+        jest.fn
+      )
+    );
+
+    expect(await screen.findByText("Result: 1")).toBeInTheDocument();
+  });
+  test("Attribute Chip Render", async () => {
+    await waitFor(() =>
+      renderDataElementsCard(
+        "attributes",
+        jest.fn,
+        testDataElements[15],
+        jest.fn
+      )
+    );
+    const resultChip = await screen.findByText("Result: 1");
+    expect(resultChip).toBeInTheDocument();
+    const closeButton = await screen.findByTestId("delete-chip-button-0");
+    expect(closeButton).toBeInTheDocument();
+    userEvent.click(closeButton);
+    expect(closeButton).not.toBeInTheDocument();
+    expect(resultChip).not.toBeInTheDocument();
+  });
+
   it("Should add new Codes", async () => {
```

Comment:
`test` and `it` both are the same but good to be consistent and follow one pattern.